pkgs/servers/irc/robustirc-bridge/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "robustirc-bridge";
+  version = "1.8";
+
+  goPackagePath = "github.com/robustirc/bridge";
+
+  src = fetchFromGitHub {
+    owner = "robustirc";
+    repo = "bridge";
+    rev = "v${version}";
+    sha256 = "12jzil97147f978shdgm6whz7699db0shh0c1fzgrjh512dw502c";
+  };
+
+  goDeps = ./deps.nix;
+
+  postInstall = ''
+    install -D $src/robustirc-bridge.1 $out/share/man/man1/robustirc-bridge.1
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Bridge to robustirc.net-IRC-Network";
+    homepage = https://robustirc.net/;
```

```suggestion
    homepage = "https://robustirc.net/";
```

https://github.com/NixOS/rfcs/pull/45